File: Data Structures & Algorithms/two-integer-sum/submission-1.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        for i in range(len(nums)):
            for j in range(len(nums)):
                if j == i:
                    j += 1
                add = nums[i] + nums[j]
                if add == target:
                    yay = list([i, j])
                    return yay
                else:
                    continue 
        
                    

File: Data Structures & Algorithms/binary-search/submission-1.cpp
class Solution {
public:
    int search(vector<int>& nums, int target) {
        int first = 0, last = nums.size()-1;
        int mid; 
        while (first <= last){
            mid = (first + last) / 2; 

            if (nums[mid] == target)
                return mid; 
            else if (nums[mid] > target)
                last = mid - 1; 
            else if (nums[mid] < target)
                first = mid +1; 
             
        }

        return -1;
    }
};
